07-uxdi-collaboration/ajax/readme.md

```diff
@@ -9,16 +9,21 @@
 - Perform POST, PUT, and DELETE requests to an API to modify data.
 
 ## Opening Framing (5/5)
-In the past couple weeks or so, we've been learning a lot about server-side requests and responses. Currently we know how to create applications with full CRUD on models in our database, and that's great. When we do that CRUD, however, it requires a page reload of some kind. It would be really nice if had some functionality on the client side of our application but still communicate with the backend without a page refresh. If only we had a client side language that was non blocking and asynchronous...
 
-(ST-WG) Think back to the first couple weeks of class, whats the difference between synchronous and asynchronous program execution?
+### PKI -
+Let's list out some technologies we've learned in this class thus far.
+
```

Comment:
right good call, i think i'll still do it, but yeah, do it or don't you can probably find a good place in your lessons for it. I just like it because its kind of a pat yourself on the back moment halfway through.